question-answering-using-openai/README.md

```diff
@@ -0,0 +1,3 @@
+# Question Answering based on Context using OpenAI GPT-3  
+
+This sample uses OpenAI GPT-3 to answer questions on a specific topic based on provided documents.
```

Comment:
Update the `README` in using following structure. 

https://github.com/ballerina-guides/integration-samples/blob/main/gcalendar-new-event-to-trello-card/README.md

Reply:
addressed in [fa5629f](https://github.com/ballerina-guides/ai-samples/pull/2/commits/fa5629f9ca6bfae66689af4f34151a7883b405d2)